### PR TITLE
Made carousel subtitle field a TextField

### LINKED ISSUE #88: Make carousel subtitles TextFields rather than CharFields

### PR DESCRIPTION
**Pull request by takeontom**
Fixes #88

### CODE REVIEW

File: glitter/blocks/carousel/templates/glitter/blocks/carouselblock_content.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
                 {% for image in carousel_images %}
                     <li data-target="#carousel-{{ content_block.id }}" data-slide-to="{{ forloop.counter0 }}" {% if forloop.first %}class="active"{% endif %}></li>
                     <h2>{{ image.title }}</h2>
-                    <p>{{ image.subtitle }}</p>
+                    <p>{{ image.subtitle | linebreaks }}</p>
```
Comment (tomkins) [inline glitter/blocks/carousel/templates/glitter/blocks/carouselblock_content.html]: Either `|linebreaksbr` for this one, or remove the `<p></p>` if you want to use linebreaks.

Comment (takeontom) [inline glitter/blocks/carousel/templates/glitter/blocks/carouselblock_content.html]: https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#linebreaks

Are you sure?

Comment (takeontom) [inline glitter/blocks/carousel/templates/glitter/blocks/carouselblock_content.html]: Ah, no you are right. :gun:

Comment (tomkins) [inline glitter/blocks/carousel/templates/glitter/blocks/carouselblock_content.html]: Yes.

If you use `|linebreaks`, you'll end up with:

```
<p><p>Text</p></p>
```

So either switch the tag, or remove the `<p>` tags.

**Pull request merged**